README.md

```diff
@@ -55,3 +55,7 @@ The following screenshots illustrate the preconfigured dashboards in this reposi
 #### Prometheus Statistics
 ![Prometheus Statistics](doc/DSEMetricsCollectorDashboardPrometheus.png)
 
+## Kubernetes Support
+These dashboards and prometheus configuration files may be transformed via the collection of Python scripts under `bin/`.
```

Comment:
Can you give a full example of how to run this with kubectl?  How does it find the hosts to scrape?

Reply:
Updated the README to include much more detailed instructions and explanations for how everything maps together.